setup.py

```diff
@@ -13,31 +13,29 @@ def read_file(filename):
 
 
 setup(
-    name='django-all-access',
+    name='django3-all-access',
```

Comment:
You are obviously welcome to publish your own version of this package under the terms of the BSD license but that shouldn't be part of this PR.